app/views/games/new.html.erb

```diff
@@ -0,0 +1,9 @@
+<div id='chessboard'>
+<% ('a'..'h').each do |letter| %>
+	<div class='board_row'> 
+	<% (1..8).each do |number| %>
+		<%= content_tag :div, 'asdf', id: "#{letter}#{number}", class: 'chessboard_space' %>
+	<% end %>
+	</div>
+<% end %>
+</div>
```

Comment:
Missing ending line break. In your editor this should look like the last line in your file is blank. There's a way to set your editor to automatically make sure of this.